.github/workflows/zeebe-version-compatibility.yml

```diff
@@ -65,10 +65,38 @@ jobs:
         with:
           name: zeebe-version-compatibility-${{ strategy.job-index }}
           path: ${{ github.workspace }}/zeebe-version-compatibility
+  current-snapshot:
+    name: Current Snapshot
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-zeebe
+        with:
+          secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secret_vault_address: ${{ secrets.VAULT_ADDR }}
+          secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
+      - uses: ./.github/actions/build-zeebe
+      - uses: ./.github/actions/build-platform-docker
+        with:
+          version: current-test
```

Comment:
:thought_balloon: Not really necessary. Normally you would build it however you want, and then set `ZEEBE_TEST_DOCKER_IMAGE` to the image name. It just so happens that `ZEEBE_TEST_DOCKER_IMAGE` defaults to `camunda/zeebe:current-test`. It might be better in fact to build it however and set that instead of relying on the default in the code :shrug:

Reply:
Maybe the `build-platform-docker` action could just set `ZEEBE_TEST_DOCKER_IMAGE` (or a more generic env var)?

Reply:
That makes sense :+1: